Makefile

```diff
@@ -264,7 +260,7 @@ prepare-and-launch-docker-to-run-tests-within: docker-test-build launch-docker-t
 
 # all additional checks not called by prove
 .PHONY: test-checkstyle-standalone
-test-checkstyle-standalone: test-shellcheck test-yaml test-critic test-js-style
+test-checkstyle-standalone: test-shellcheck test-yaml test-critic
```

Comment:
It would be good to have some alternative (not necessary a Makefile target, a script is fine as well) and documentation how to call it.

Reply:
The main question here is how we want to handle the `node` dependency (and `npm install` call to install the npm modules).

Reply:
We can bail out if `npm` is not available. We don't have special support for `prove` or other tools in the Makefile either

Reply:
`prove` ships with Perl though.

Reply:
Maybe let's just add a tidy script under `tools/` which invokes whatever it needs to invoke and prints an error message with installation instructions if it isn't there. It would be good if it would check for locally installed packages first and for globally installed packages second. (So it isn't required to install as root.)

For the CI it might make sense to install the dependency when creating the container.

`npm` itself could be added as dependency of `openQA-devel`.

Reply:
The script idea makes sense for local development. About CI we don't really have to worry since that's running as a GitHub Action with an Ubuntu container and `actions/setup-node` anyway (for the GitHub annotations).